ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -617,8 +617,8 @@ tests:
     - ref: openshift-extended-web-tests
     - ref: cucushift-e2e-report
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-ovn-hypershift-guest-p2-f28-destructive
-  cron: 59 11 12 * *
+- as: aws-ipi-ovn-hypershift-guest-p2-f21-destructive
```

Comment:
The change for destructive test is expected ?

Reply:
We want to make sure it could be run within one month. It will support us to compare data with 4.13.

Reply:
f28, runs on the 12th day of the month, at 11:59,
f21, runs on the 5th day of the month, at 4:38
I didn't see it make any differences.

Reply:
So can I assume that f21 and f28 have no difference in test frequency?

Reply:
Exactly. f21 means every 21 days, we do not have two 21 in one month.

Reply:
okay, got it, But at present, we do not define `f28` for Hypershift, so for the consistency of code, we have changed to `f21`

Reply:
@LiangquanLi930 Sorry, I just not check clearly. This file need to update to f14 as it's handling guest cluster E2E testing. We will need those data to support OSD FM testing and progressive testing.

Reply:
Guest clusters E2E testing is most important to our Managed-hypershift release.